buildbot/osuosl/master/config/builders.py

```diff
@@ -910,6 +910,8 @@
                     extra_cmake_args=[
                         "-DCMAKE_C_FLAGS='-march=cascadelake'",
                         "-DCMAKE_CXX_FLAGS='-march=cascadelake'",
+                        "-DCMAKE_C_COMPILER='icx.EXE'",
```

Comment:
You do not need quotation marks around the executable name here and in the line below. Quotation marks wrapping could be tricky on Windows, better to avoid unneeded all together.

Reply:
Thank you so much, @gkistanova.